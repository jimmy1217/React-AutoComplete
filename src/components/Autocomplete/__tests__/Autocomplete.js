import React from 'react';
import renderer from 'react-test-renderer'
import { Autocomplete } from './../../index.js'

describe('render 測試', () => {
  it('Autocomplete Snapshot', () => {
    const component = renderer.create(<Autocomplete />)
    const json = component.toJSON()
    expect(json).toMatchSnapshot()
  })
})