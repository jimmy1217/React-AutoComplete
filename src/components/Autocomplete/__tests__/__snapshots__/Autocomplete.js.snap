// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 測試 Autocomplete Snapshot 1`] = `
<div
  className="rj_autocompleteContent"
>
  <div
    className="icon_addon fw-900"
  >
    <span
      className="defaultSearch"
    >
      ⚲
    </span>
  </div>
  <input
    autoFocus={false}
    className="rj_autocomplete"
    disabled={false}
    onChange={[Function]}
    onClick={[Function]}
    onKeyDown={[Function]}
    placeholder="請搜尋想找的項目"
    readOnly={false}
    type="text"
    value=""
  />
  <div
    className="icon_addon fw-900 mr-10 hidden"
  >
    <span
      className="defaultSearch"
    >
      ⚲
    </span>
  </div>
  <hr />
  <hr
    className="focus-border"
  />
  <div
    className="overlayCancel hidden"
    onClick={[Function]}
  />
  <div
    className="rj_list rj_list_hidden"
  >
    <ul>
      <li
        className="result active"
        onClick={[Function]}
        onMouseOver={[Function]}
      >
        apple
      </li>
      <li
        className="result"
        onClick={[Function]}
        onMouseOver={[Function]}
      >
        banana
      </li>
    </ul>
  </div>
</div>
`;
